docsite/index.html

```diff
@@ -2072,10 +2072,9 @@ <h5>Usage Sample</h5>
       <article class="font-sizes">
         <h4>Font Families</h4>
         <pre class="language-css"><code>
-          --font-sans: system-ui, -apple-system, Segoe UI, Roboto, Ubuntu, Cantarell, Noto Sans, sans-serif;
-          --font-serif: ui-serif, serif;
-          --font-mono: Dank Mono, Operator Mono, Inconsolata, Fira Mono, ui-monospace, SF Mono, Monaco, Droid Sans Mono, Source Code Pro, monospace;
-        </code></pre>
+          --font-sans: ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
seems strange to put emoji fonts into this list?

Reply:
If none of the earlier fonts in the stack include an emoji, the system will use the emoji font to display the symbol

Reply:
ic, jumps into the fallback line 👍🏻 
shouldn't those be in the serif font too then?

Reply:
I'm not sure honestly